src/pages/index.html

```diff
@@ -4,16 +4,380 @@
+
+
+
+
+</div>
+<div class="Payment">
+
+      
+  <div class="easiest  ">
+  
+  
+  
+  
+  <div class="card1" >
+  
+      <h3> Payment is the easiest with many options  </h3>
+      <p> Credit cards are easily the most secure and safe payment method to use when you shop online.  </p>
+  
+    <div class="Manage">
+      <div class="  d-flex  align-items-center">
+        <i class="fa-solid fa-check"></i>
+        <p>    Accept any method of payment</p>
+      </div>
+      <div class="  d-flex r align-items-center ">
+        <i class="fa-solid fa-check"></i>
+        <p>   Apply discounts and issue refunds</p>
+      </div>
+    </div>
+ <a href="#">  Make your first sale <i class="fa-solid fa-arrow-right"></i> </a>
+    
+    </div>
+  
+    <div class="card2" >
+      <div class="shap">
+<img src="../assets/images/shape.png" alt="">
+
+      </div>
+      <h4> Payment Method </h4>
+
+      <div class="card0">
+      <div class="bank d-flex">
+        <img src="../assets/images/Icon (7).svg" alt="">
+        <div class="sdd ">
+          <div>
+          <h5> Add Card  </h5>
+          <p>Add a card of any bank  </p>
         </div>
-    </section>
+        <label class="checkbox-container">
+          <input type="checkbox">
+          <span class="checkmark"></span>
+        </label>
+        </div>
+      </div>
+      <div class="bank d-flex">
+        <img src="../assets/images/Icon (7).svg" alt="">
+        <div class="sdd    ">
+          <div >
+            <h5> Add Crypto Wallet  </h5>
+            <p>Connect your crypto wallet  </p>
+          </div>
+      
+          <label class="checkbox-container">
+            <input type="checkbox">
+            <span class="checkmark"></span>
+          </label>
+      </div>
+    </div>
+      <div class="bank d-flex">
+      <img src="../assets/images/Icon (7).svg" alt="">
+      <div class="sdd ">
+        <div >
+
+        <h5> Add Unipay Account  </h5>
+      <p>Hold all payments via the app </p>
+    </div>
+    <label class="checkbox-container">
+      <input type="checkbox">
+      <span class="checkmark"></span>
+    </label>
+  </div>
+
+  </div>
+      <div class="visa">
+        <i class="  fa-brands fa-cc-visa"></i> 
+        <i class="fa-brands fa-cc-mastercard"></i>
+                  <i class=" fa-brands fa-cc-visa">
+          </i><i class="fa-brands fa-cc-visa"></i>   
+          
+          
+        </div>
+      </div>
+    </div>
+
+  </div>
+  </div>
+  <div class="container">
+
+<div class="Resource-Management">
+  <div class="Management ">
+    <div class="box">
+      <img src="../assets/images/Layer 1.svg" alt="">
+      <h5 >  Resource Management</h5>
+      <p >Resource management is the process of pre-planning, 
+        scheduling, and allocating
+         your resources to maximize efficiency.</p>
+    </div> <div class="box">
+      <img src="../assets/images/Layer 2.svg" alt="">
+      <h5 >  Resource Management</h5>
```

Comment:
![Capture](https://github.com/shaban011147/landing/assets/106445705/389b5386-f8d8-4da6-aed0-7bbf6937a09d)
1- space between h5 (Resource Management) and p tag not same in design
2- space vertical between div (box) not same in design

Reply:
done